src/dashboard/infrastructure/integrations/site-kit.php

```diff
@@ -109,6 +120,24 @@ public function is_onboarded() {
 		return ( $this->is_site_kit_installed() && $this->is_setup_completed() && $this->is_connected() );
 	}
 
+	/**
+	 * Can user read site kit data.
+	 *
+	 * @param string $slug The slug of the data.
+	 * @return bool If the user can read the data.
+	 */
+	private function can_read_data( $slug ) {
+		$dashboard_sharing    = \get_option( 'googlesitekit_dashboard_sharing' );
+		$search_console_roles = isset( $dashboard_sharing[ $slug ] ) ? $dashboard_sharing[ $slug ]['sharedRoles'] : [];
```

Comment:
Seems you started with search console. But now it is also for analytics 😉 
Please adjust these names, and below in options too